py/picca/tests/delta_extraction/expected_flux_tests.py

```diff
@@ -131,8 +131,9 @@ def test_dr16_expected_flux_compute_continuum(self):
         f.write("# thingid cont[0] ... cont[N]\n")
         for forest in data.forests:
             f.write(f"{forest.los_id} ")
-            for item in forest.continuum:
-                f.write(f"{item} ")
+            if forest.continuum is not None:
```

Comment:
This is to prevent the type error `TypeError: 'NoneType' object is not iterable` when the forest continuum fitting had some problems (and thus `forest.continuum = None`). This could happen in the tests if we, at some point, change some of the parameters. I can't recall this happening but I must have seen this and thought to change it in case